src/Utils/messages-media.ts

```diff
@@ -4,7 +4,7 @@ import { exec } from 'child_process'
 import * as Crypto from 'crypto'
 import { once } from 'events'
 import { createReadStream, createWriteStream, promises as fs, WriteStream } from 'fs'
-import type { IAudioMetadata } from 'music-metadata'
+import type { IAudioMetadata } from 'music-metadata' with { 'resolution-mode' : 'require' }
```

Comment:
this is an import type only. Its necessary to use resolution-mode?

Reply:
Without it:

```
> baileys@6.7.16 build:tsc
> tsc

src/Utils/messages-media.ts:7:37 - error TS1541: Type-only import of an ECMAScript module from a CommonJS module must have a 'resolution-mode' attribute.
  To convert this file to an ECMAScript module, change its file extension to '.mts', or add the field `"type": "module"` to ...
```
but I think it better to use `resolution-mode: import`, given it is from an ESM module